types/react-native/index.d.ts

```diff
@@ -43,6 +43,7 @@
 //                 Mateusz Wit <https://github.com/MateWW>
 //                 Luna Wei <https://github.com/lunaleaps>
 //                 Saad Najmi <https://github.com/saadnajmi>
+//                 Lewis Yearsley <https://github.com/lewisyearsley>
```

Comment:
Are you intentionally adding yourself as an owner of this package? If you're looking for credit, that is handled by the git history (this is if you want to review and approve changes to this package).

Reply:
Yes